### PR TITLE
Task ys/j206 fix comment timeline with

### PR DESCRIPTION
**Pull request by yaroslav207**

### CODE REVIEW

File: frontend/src/components/episode/components/timeline-item/styles.module.scss [hunk 1]
```diff
@@ -32,4 +34,10 @@
     margin-right: 10px;
     border-radius: 50%;
   }
+  .timelineItemInfo{
+    width: 200px;
+  }
```
Comment (artem-totality) [inline frontend/src/components/episode/components/timeline-item/styles.module.scss]:
```suggestion
  .timelineItemInfo{
    div {
      overflow: hidden;
      text-overflow: ellipsis;
      white-space: nowrap;
      max-width: 200px;
    }
  }
```

File: frontend/src/components/episode/components/timeline-item/timeline-item.tsx [hunk 2]
```diff
@@ -40,7 +51,7 @@ const TimelineItem: React.FC<Props> = ({
         onClick={handleGoToItemTimestamp}
         className={styles.timelineItemBtn}
       />
-      <div className={styles.timelineItemContent}>
+      <div style={{ left: `${offsetBlockComment}px` }} className={styles.timelineItemContent}>
```
Comment (artem-totality) [inline frontend/src/components/episode/components/timeline-item/timeline-item.tsx]:
```suggestion
      <div style={commentOffset < 50 ? { left: 0 } : { right: 0 }} className={styles.timelineItemContent}>
```
This change with suggested changes in style will be more simple (or some variants of them)

File: frontend/src/components/episode/components/timeline-item/helpers/get-offset-comment-block/get-offset-comment-block.helper.ts [hunk 3]
```diff
@@ -0,0 +1,7 @@
+const getOffsetCommentBlock = (commentOffset: number, containerWidth: number, commentBlockWidth: number): number => {
```
Comment (what1s1ove) [inline frontend/src/components/episode/components/timeline-item/helpers/get-offset-comment-block/get-offset-comment-block.helper.ts]: we don't want it
just need styles that we want

File: frontend/src/components/episode/components/timeline-item/timeline-item.tsx (hunk 2 above)
Comment (what1s1ove) [reply frontend/src/components/episode/components/timeline-item/timeline-item.tsx]: We don't need it. Just add styles that we want

**Review by what1s1ove — CHANGES_REQUESTED** [review]

**Review by what1s1ove — APPROVED** [review]

**Pull request merged**